packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/synthesize-resources.ts

```diff
@@ -275,17 +283,16 @@ export const updateUserPoolGroups = async (context: $TSContext, resourceName: st
     ensureDirSync(userPoolGroupFolder);
     JSONUtilities.writeJson(path.join(userPoolGroupFolder, 'user-pool-group-precedence.json'), updatedUserPoolGroupList);
 
-    context.amplify.updateamplifyMetaAfterResourceUpdate(AmplifyCategories.AUTH, 'userPoolGroups', 'userPoolGroups', {
-      service: 'Cognito-UserPool-Groups',
-      providerPlugin: 'awscloudformation',
-      dependsOn: [
-        {
-          category: AmplifyCategories.AUTH,
-          resourceName,
-          attributes: ['UserPoolId', 'AppClientIDWeb', 'AppClientID', 'IdentityPoolId'],
-        },
-      ],
-    });
```

Comment:
nice find, but I'm curious how this is going to affect the rest of the application. @akshbhu , you recently modified this, can you do a sanity check on this PR?

in reference to your previous modification: https://github.com/aws-amplify/amplify-cli/commit/9fa8b4d86c1e8cf0e562d3d6c61b0bd3bae329b4#diff-4740f906c10f1d624337cd06a9e78ee0d744dc85c51f92dcf0774c6d05e2c942R248